module/PowerNSX.psm1

```diff
@@ -26657,6 +26657,77 @@ function Add-NsxServiceGroupMember {
     end {}
 }
 
+function Remove-NsxServiceGroupMember {
+
+    <#
+    .SYNOPSIS
+    Removes a single Service, numerous Services, or a Service Group to a Service
+    Group
+    
+    .DESCRIPTION
+    Removes the defined Service or Service Group to an NSX Service Groups. Service
+    groups contain a mixture of selected ports to represent a potential
+    grouping of like ports.
+    
+    This cmdlet removes the defined Services or Service Groups within a Service
+    Group for specific or all Service Groups 
+    
+    .EXAMPLE
+    Get-NsxServiceGroup Heartbeat | Remove-NsxServiceGroupMember -Member $Service1
+    
+    Remove all saved Service Group Members named Heartbeat in Service Group in $Service1
+    
+    Get-NsxServiceGroup Service-Group-4 | Remove-NsxServiceGroupMember $Service1,$Service2 -Confirm:$false
+    
+    Remove all saved Service Group Members named Heartbeat in Service Group in $Service1 and $Service2 without prompting for confirmation
+    
+    #>
+    
+    param (
+        #Mastergroup removed from Get-NsxServiceGroup
+        [Parameter (Mandatory=$true,ValueFromPipeline=$true)]
+            [ValidateScript({ ValidateServiceGroup $_ })]
+            [System.Xml.XmlElement]$ServiceGroup,
+        [Parameter (Mandatory=$False)]
+            #Prompt for confirmation.  Specify as -confirm:$false to disable confirmation prompt
+            [switch]$Confirm=$true,
+        [Parameter (Mandatory=$true,Position=1)]
+            [ValidateScript({ ValidateServiceOrServiceGroup $_ })]
+            #The [] in XmlElement means it can expect more than one object!
```

Comment:
Avoid programmer comments in the param block - these are visible to the user in get-help and should be used to explain what the param is.  you can tell the user that you accept a collection if you want but do it in a user centric way.